dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/controller/ProjectParameterControllerTest.java

```diff
@@ -47,12 +48,17 @@ public class ProjectParameterControllerTest {
     @Test
     public void testCreateProjectParameter() {
         User loginUser = getGeneralUser();
+        Result successResult = getSuccessResult();
+        successResult.setData(getProjectParameter());
 
         Mockito.when(projectParameterService.createProjectParameter(Mockito.any(), Mockito.anyLong(), Mockito.any(),
-                Mockito.any(), Mockito.any())).thenReturn(getSuccessResult());
+                Mockito.any(), Mockito.any())).thenReturn(successResult);
         Result result = projectParameterController.createProjectParameter(loginUser, 1, "key", "value",
                 DataType.VARCHAR.name());
         Assertions.assertEquals(Status.SUCCESS.getCode(), result.getCode());
+
+        ProjectParameter projectParameter = (ProjectParameter) result.getData();
+        Assertions.assertEquals(1, projectParameter.getOperator());
```

Comment:
```suggestion
        Assertions.assertEquals(loginUser.getId(), projectParameter.getOperator());
```

Reply:
Does this UT have any help to this case? You need to add UT to `ProjectParameterService` and avoid mocking the result.

Reply:
> Does this UT have any help to this case? You need to add UT to `ProjectParameterService` and avoid mocking the result.
Useful because if created successfully, it returns projectParameter. We can verify if the modify user has a value to determine if the addition was successful

Reply:
If you revert the change of `projectParameterService` the case still pass.

Reply:
> If you revert the change of `projectParameterService` the case still pass.

You've undone the modification, so isn't the operator now null, or are we not on the same page?

Reply:
I means since the `projectParameterService.createProjectParameter` has been mocked, so the test case is meaningless, it can just test the api call is work but cannot test the logic in `projectParameterService`.

Reply:
> I means since the `projectParameterService.createProjectParameter` has been mocked, so the test case is meaningless, it can just test the api call is work but cannot test the logic in `projectParameterService`。

The adjustment has been made, this should be suitable now